src/helpers/ci.ts

```diff
@@ -505,23 +505,31 @@ export const getCISpanTags = (): SpanTags | undefined => {
   if (env.BUDDY) {
     const {
       BUDDY_PIPELINE_NAME,
+      BUDDY_PIPELINE_ID,
       BUDDY_EXECUTION_ID,
       BUDDY_SCM_URL,
       BUDDY_EXECUTION_BRANCH,
+      BUDDY_EXECUTION_TAG,
       BUDDY_EXECUTION_REVISION,
       BUDDY_EXECUTION_URL,
       BUDDY_EXECUTION_REVISION_MESSAGE,
+      BUDDY_EXECUTION_REVISION_COMMITTER_NAME,
+      BUDDY_EXECUTION_REVISION_COMMITTER_EMAIL,
     } = env
 
     tags = {
       [CI_PROVIDER_NAME]: CI_ENGINES.BUDDY,
-      [CI_PIPELINE_ID]: BUDDY_EXECUTION_ID,
-      [CI_PIPELINE_URL]: BUDDY_EXECUTION_URL,
+      [CI_PIPELINE_ID]: `${BUDDY_PIPELINE_ID}/${BUDDY_EXECUTION_ID}`,
       [CI_PIPELINE_NAME]: BUDDY_PIPELINE_NAME,
+      [CI_PIPELINE_NUMBER]: `${BUDDY_EXECUTION_ID}`, // gets parsed to int again later
```

Comment:
can you clarify where it gets parsed?

Reply:
Here: https://github.com/DataDog/datadog-ci/blob/787f2f1dbececd218d9d2ddab639fe814ecd7ead/src/helpers/ci.ts#L582

I'll update the comment.